src/include/function/gds/gds_frontier.h

```diff
@@ -155,45 +118,46 @@ class KUZU_API SparseFrontier {
  *
  * However, this is not necessary and the caller can also use this to represent a single frontier.
  */
-class KUZU_API PathLengths : public GDSFrontier {
```

Comment:
Should this be renamed to `DenseFrontier` now? I don't follow why `PathLengths` is a generic name, as we should have cases where the frontier has nothing to do with path?

Reply:
Maybe but I'm less motivated to do the renaming at this point.